packages/faceted-search/src/queryClient/tql.js

```diff
@@ -23,9 +23,13 @@ const prepareBadges = flow([removeBadgesWithEmptyValue, getBadgesQueryValues]);
 const getTqlClassOperatorsDictionary = query => ({
 	contains: query.contains,
 	'=': query.equal,
+	in: query.in,
 });
 
 const formatValue = value => {
+	if (Array.isArray(value)) {
+		return value.map(val => val.id).join(',');
```

Comment:
Here, if i get a, empty array as value, we return `''` which can be get a TQL query like, for example, `connection.type in []`.

Are we ok with that? Label in facet show a "all", be i think query return 0 element no?

Reply:
good point